.github/workflows/auto-publish.yml

```diff
@@ -12,3 +12,7 @@ jobs:
       - uses: w3c/spec-prod@v2
         with:
           GH_PAGES_BRANCH: gh-pages
+          W3C_ECHIDNA_TOKEN: ${{ secrets.ECHIDNA_TOKEN }}
+          W3C_WG_DECISION_URL: https://lists.w3.org/Archives/Public/public-secondscreen/2022Apr/0007.html
+          W3C_BUILD_OVERRIDE: |
+            Status: WD
```

Comment:
That depends on the underlying editing tool that the spec uses.

- With ReSpec, this would be [`specStatus`](https://respec.org/docs/#specStatus).
- With Bikeshed, this is [`Status`](https://tabatkins.github.io/bikeshed/#metadata-status).

See also the [spec-prod documentation](https://w3c.github.io/spec-prod/#gh_pages_build_override).

In the end, I always forget about this and even when I don't forget, I usually pick up the wrong parameter name... I think I got it right for once ;)

Note that, at worst, publication to /TR will fail (W3C's publication system will reject the spec if its metadata are not the expected ones. In our case, it will prevent publication of anything that does not present itself as a Working Draft).

Reply:
Okay, thank you!